.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, dev, dev-3.0.0, release/3.0.0 ]
+  pull_request:
+    branches: [ master, dev, dev-3.0.0, release/3.0.0 ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Install VirtoCommerce.GlobalTool
+      run: dotnet tool install --global VirtoCommerce.GlobalTool --version 3.0.0-beta0010
```

Comment:
Should we install the specific version? 
--version 3.0.0-beta0010
I think to avoid changes with new Global tool versions it is better to use the latest.

Reply:
It doesn't work because it's still in a pre-release.

Reply:
I see